analyses/snv-callers/vaf_cutoff_experiment.Rmd

```diff
@@ -0,0 +1,284 @@
+---
+title: "VAF Cutoffs and TMB Calculation"
+output: 
+  html_notebook:
+    toc: TRUE
+    toc_float: TRUE
+author: C. Savonen for ALSF CCDL
+date: 2019
+---
+
+Purpose: How do the TMB calculations change with adjustments to the VAF cutoff
+filter? 
+
+### Summary of Findings:
+
+#### Usage
+
+To run this from the command line, use:
+```
+Rscript -e "rmarkdown::render('analyses/snv-callers/vaf_cutoff_experiment.Rmd', 
+                              clean = TRUE)"
+```
+
+_This assumes you are in the top directory of the repository._
+
+## Setup
+
+#### Packages and functions
+
+Read in set up script.
+
+```{r}
+if (!("GGally" %in% installed.packages())) {
+  install.packages("GGally")
+}
+if (!("ggupset" %in% installed.packages())) {
+  install.packages("ggupset")
+}
+# Magrittr pipe
+`%>%` <- dplyr::`%>%`
+```
+
+Set up output directories. 
+
+```{r}
+base_results_dir <- "results"
+base_plots_dir <- "plots"
+```
+
+Make new directories for the comparison analysis.
+
+```{r}
+compare_results_dir <- file.path(base_results_dir, "comparisons", "vaf_cutoff")
+compare_plots_dir <- file.path(base_plots_dir, "comparisons", "vaf_cutoff")
+
+# Make caller specific plots folder
+if (!dir.exists(compare_results_dir)) {
+  dir.create(compare_results_dir)
+}
+# Make caller specific plots folder
+if (!dir.exists(compare_plots_dir)) {
+  dir.create(compare_plots_dir)
+}
+```
+
+Function for creating a TMB correlation matrix amongst the callers for a particular cutoff.
+
+```{r}
+cor_matrix_plot <- function(cutoff) {
```

Comment:
I like the idea of putting these custom functions in their own script in a `util` directory.

Reply:
@jashapiro What do you think about this idea? I'm kinda leaning toward it.